lib/src/main/java/uk/gov/di/ipv/cri/address/library/persistence/item/AddressSessionItem.java

```diff
@@ -15,13 +17,15 @@
     private String clientId;
     private String state;
     private URI redirectUri;
+    private List<CanonicalAddressWithResidency> addresses;
```

Comment:
Why not 

`private List<CanonicalAddressWithResidency> addresses = new ArrayList<>();`